Sources/Operators.swift

```diff
@@ -241,7 +241,7 @@ public func <~~ (key: String, json: JSON) -> [URL]? {
 /**
 Encode custom operator.
 */
-infix operator ~~> { associativity left precedence 150 }
+infix operator ~~> : MultiplicationPrecedence
```

Comment:
Yeah, you are right, it would be `AdditivePrecedence` to keep the same.

I believe it makes little difference, I think `CastingPrecedence` is more expressive. 
We aren't supposed to be being mixed with math operators anyway, and we make it clear to anyone reading that we need to have precedence over casting and expressions like `let id = "id" <~~ json as Int?` work without trying to cast `json`.